analyzers/tests/SonarAnalyzer.UnitTest/Rules/AvoidLambdaExpressionInLoopsInBlazorTest.cs

```diff
@@ -37,6 +37,12 @@ public class AvoidLambdaExpressionInLoopsInBlazorTest
             .WithAdditionalFilePath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Product))
             .Verify();
 
+    [TestMethod]
+    public void AvoidLambdaExpressionInLoopsInBlazor_BlazorLoopsWithNoBody() =>
+        builder.AddPaths("AvoidLambdaExpressionInLoopsInBlazor.LoopsWithNoBody.razor")
+            .WithAdditionalFilePath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Product))
+            .Verify();
```

Comment:
I added another test method, the `DiagnosticVerifier` does not accept more than one file to have issues when the [SourceTree](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs#L220C19-L220C131) does not exist, which is the case for `.razor`.